frontend2/src/contexts/CurrentTeamProvider.tsx

```diff
@@ -44,7 +51,9 @@ export const CurrentTeamProvider: React.FC<{ children: React.ReactNode }> = ({
   }, [authState, episodeId]);
 
   return (
-    <CurrentTeamContext.Provider value={{ ...teamData, leaveMyTeam }}>
+    <CurrentTeamContext.Provider
```

Comment:
Looks like `refreshTeam` sets `teamState` to `IN_TEAM` no matter what? Is this only callable if the user is already in a team?

Reply:
Yeah, this function should only be called when the user is already on a team and has received an updated TeamPrivate object from some api call

Reply:
I'll add a comment describing this